examples/xml-access/xml_access.bal

```diff
@@ -14,30 +14,30 @@ public function main() {
                 <!--Price: $10-->
                 </book>`;
     
-    // You can access child XML items using xml step expressions.
+    // You can access child XML items using XML step expressions.
     io:println(bookXML/<author>/<fname>);
 
-    // Accessing a non-existing child will return empty xml sequence.
+    // Accessing a non-existing child will return an empty `xml` sequence.
     io:println(bookXML/<ISBN>/<code>);
 
     // You can also retrieve attributes of the resulting child XML.
     io:println(bookXML/<author>/<fname>.title);
 
-    // You can match descendent elements using following steping access syntax.
+    // You can match descendant elements using the following stepping access syntax.
     io:println(bookXML/**/<fname>);
 
-    // Select all children using bellow syntax.
+    // Select all children using the bellow syntax.
     io:println(bookXML/*);
 
-    // And select all children elements using bellow syntax.
+    // Select all children elements using the bellow syntax.
     io:println(bookXML/<*>);
 
-    // Select all children belong to specific namespace.
+    // Select all the children belonging to a specific namespace.
```

Comment:
```suggestion
    // Select all the children elements belonging to a specific namespace.
```

Reply:
Children `elements` are selected using <*> syntax
This select 'elements' and non element items, hence we can omit the `elements` word right?

Reply:
+1